src/components/SelectAccount/index.js

```diff
@@ -62,7 +62,8 @@ type Props = {
   t: T,
 }
 
-const getOptionValue = account => account.id
+const getOptionValue = account =>
+  `${account.currency.ticker}|${account.currency.name}|${account.name}`
```

Comment:
does the `|` creates a problem for the search engine? i wonder if he consider a word. For instance, `Ethereum Classic`, if you search Classic (and your account name does not have classic in it) does it work?

because I wonder if `ETC|Ethereum Classic|foo` would make some problem

Intuitively i would have used a space

Reply:
I doubt it would cause any issues, the filtering is done using a simple `indexOf(input) > -1` after some cleanup. As long as the input string is somewhere in the concat we built it should be fine. If you feel more comfortable with spaces feel free to commit 🤷‍♂️ 

https://github.com/JedWatson/react-select/blob/master/src/filters.js#L40